source/FAST/Visualization/WindowWidget.cpp

```diff
@@ -17,6 +20,28 @@ void WindowWidget::keyPressEvent(QKeyEvent *event) {
         view->keyPressEvent(event);
 }
 
+void WindowWidget::dragEnterEvent(QDragEnterEvent *event) {
+    Reporter::info() << "\nDrag event..." << Reporter::end();
+    if (event->mimeData()->hasUrls()) {
+        event->acceptProposedAction();
+    }
+}
+
+
+void WindowWidget::dropEvent(QDropEvent *event) {
+    std::cout << "\nDrop event..." << std::endl;
```

Comment:
Don't use cout please, use reporter instead. \n here is not necessary
```suggestion
    Reporter::info() << "Drop event received in window widget" << Reporter::end();
```